modules/ROOT/pages/anypoint-platform-cli-commands.adoc

```diff
@@ -1638,8 +1745,10 @@ To redeploy an app and set a logging level or turn on Insight event tracking, se
 > runtime-mgr standalone-application describe-json [options] <identifier>
 ----
 
-This command describes the standalone (on-premises) application passed in `identifier` as a raw JSON response. +
-This command does not take any options, except for the default ones: `--help`, `-f`/`--fields` and `-o`/`--output`.
+This command describes the standalone (on-premises) application passed in `identifier` as a raw JSON response.
```

Comment:
What do we mean by a standalone application? We have standalone instances of Mule. Do they mean applications deployed on-premises?

Reply:
Yep!